src/sql/azdata.proposed.d.ts

```diff
@@ -432,6 +432,24 @@ declare module 'azdata' {
 		azurePortalEndpoint?: string;
 	}
 
+	/**
+	 * An interface for receiving diagnostic data from a provider.
+	 */
+	export interface Diagnostics {
```

Comment:
should the `Diagnostics` interface be contained in the `diagnostics` namespace?  or is this following the pattern VS Code types are using?

Reply:
This is following the resourceProvider and resources namespace along with Dashboard and DashboardWebView